app/static/js/alpheios/annotate.enter.js

```diff
@@ -314,8 +318,8 @@ function save_data(){
     });
 }
 
-
-
+//Why doesn't the above ajax post to save_data?  Why can't we do the built in form thing and have it automatically post to save data?
+//Is it just because that's how perseids stuff does it?
```

Comment:
@annakrohn I agree this is a little wonky.  One difference here is that when used with perseids as a back-end, the save_data post is to a different app than the one that is serving the input form, whereas, as it's being used here for digital milliet, the save is to the same app -- it should probably be separated and the post go to digital milliet and not the perseids-client-app -- this is one of the things I actually wanted to rework a bit before merging into master